_data/changelogs/component-file-input.yml

```
@@ -2,6 +2,12 @@ title: File input
 type: component
 changelogURL:
 items:
+  - date: NNNN-NN-NN
+    summary: Removed disabled file input variant preview.
+    summaryAdditional: 
```

Needs a quick addition:
```suggestion
    summaryAdditional: 
    affectsGuidance: true
```